crypto/asn1/asn1_gen.c

```diff
@@ -699,7 +699,10 @@ static ASN1_TYPE *asn1_str2type(const char *str, int format, int utype)
             atmp->value.asn1_string->length = rdlen;
             atmp->value.asn1_string->type = utype;
         } else if (format == ASN1_GEN_FORMAT_ASCII)
```

Comment:
Add `{}` around this `if`. Otherwise the following `else if` becomes ambiguous.

Reply:
Yes, sure. Thanks. This is my oversight